### PR TITLE
Dynamic input RPM link by region setting

### PR DESCRIPTION
**Pull request by XinRanZhAWS**
Dynamic input RPM link by region setting, noted that IAD have it's specific form of link

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: .github/workflows/appsignals-e2e-ec2-test.yml [hunk 1]
```diff
@@ -34,6 +33,15 @@ jobs:
         with:
           fetch-depth: 0
 
+      - name: Set Parameter APP_SIGNALS_CW_AGENT_RPM
```
Comment (majanjua-amzn) [inline .github/workflows/appsignals-e2e-ec2-test.yml:36]: nit: Make this more obvious what it is doing, for example "Set CW Agent RPM environment variable"

**Review by majanjua-amzn — COMMENTED** [review]
LGTM, just a small comment. Also, make sure to run a test to make sure it's working and doesn't break the canary. You can do so via the Actions page and run the workflow on manually and choose your fork as the branch.

File: .github/workflows/appsignals-e2e-ec2-test.yml (hunk 1 above)
Comment (XinRanZhAWS) [reply .github/workflows/appsignals-e2e-ec2-test.yml:36]: Changing based on that

**Review by majanjua-amzn — APPROVED** [review]
LGTM, tested the change in the private repo: https://github.com/aws-observability/aws-apm-java-instrumentation/actions/runs/719967825

**Review by PaurushGarg — APPROVED** [review]

**Pull request merged**